slash-commands/utility/clearinfractions.js

```diff
@@ -0,0 +1,64 @@
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {promisePool} = require('../../config/db');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('clearinfractions')
+    .setDescription('Removes all infractions from a user')
+    .addUserOption((option) =>
+      option.setName('user').setDescription('The user').setRequired(true)
+    ),
+  async execute(interaction) {
+    const targetUser = await interaction.options.getUser('user');
+    try {
+      const hasInfractions = await checkForInfractions(targetUser.id);
+      if (!hasInfractions) {
+        return interaction.reply('This user does not have any infractions.');
+      }
+      await deactivateInfractionsDB(interaction, targetUser);
+      await sendToAuditLogsChannel(interaction, {
+        color: '#0099ff',
+        titleMsg: `${interaction.user.tag} cleared all infractions for ${targetUser.username}#${targetUser.discriminator}`,
```

Comment:
Are we able to just use `$targetUser.tag` here instead of the username + discriminator?

Reply:
Good idea.